pyproject.toml

```diff
@@ -0,0 +1,51 @@
+[build-system]
+requires = ["setuptools>=62.0", "setuptools-git-versioning<2"]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "diffpy.pdfmorph"
+dynamic=['version']
+authors = [
+  { name="Simon J.L. Billinge group", email="simon.billinge@gmail.com" },
+]
+maintainers = [
+  { name="Simon J.L. Billinge group", email="simon.billinge@gmail.com" },
+]
+description = "Tools for manipulating and comparing PDF profiles."
+keywords = ["diffpy PDF"]
+readme = "README.rst"
+requires-python = ">=3.10"
+classifiers = [
+        'Development Status :: 4 - Beta',
+        'Environment :: Console',
+        'Intended Audience :: Science/Research',
+        'License :: OSI Approved :: BSD License',
+        'Operating System :: MacOS :: MacOS X',
+        'Operating System :: Microsoft :: Windows',
+        'Operating System :: POSIX',
+        'Operating System :: Unix',
+        'Programming Language :: Python :: 3.10',
+        'Programming Language :: Python :: 3.11',
+        'Programming Language :: Python :: 3.12',
+	'Topic :: Scientific/Engineering :: Chemistry',
+        'Topic :: Scientific/Engineering :: Physics',
+]
+
```

Comment:
we may need to add scripts or entry points I think.  Yucong ran into this with labpdfproc that it wouldn't run, then I added an entry point and resinstalled and it ran.